judgels-backends/jophiel/jophiel-app/src/integTest/java/judgels/jophiel/user/UserStoreIntegrationTests.java

```diff
@@ -91,6 +94,14 @@ void crud_flow() {
 
         Page<User> users = store.getUsers(Optional.empty(), Optional.empty(), Optional.empty());
         assertThat(users.getPage()).containsExactly(budi, foxtrot, nano, user);
+
+        Map<String, User> userMap = store.getUsersByUsername(
+                new HashSet<>(Arrays.asList("budi", "foxtrot", "notexists"))
```

Comment:
This codebase prefers Guava-based collection:

```suggestion
                ImmutableSet.of("budi", "foxtrot", "notexists")
```